### PR TITLE
Override Features

### PR DESCRIPTION
**Pull request by pdr9rc**
Introducing the overrides feature that was discussed with Thomas. Also includeds Cloudtrail to ECS and Exported Fields mappings and EC2 VM export rule for AWS.

### CODE REVIEW

File: rules/cloud/aws_ec2_vm_export_failure.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - attack.t1537
+level: low
+logsource:
+  service: cloudtrail
+detection:
+  selection:           
+    eventName: 'CreateInstanceExportTask'
+    eventSource: 'ec2.amazonaws.com'
+  filter1:
+    errorMessage: '*'
```
Comment (Neo23x0) [inline rules/cloud/aws_ec2_vm_export_failure.yml:22]: @thomaspatzke : is this '*' correct or do we have to use 'null' and negate it in the condition?

Comment (thomaspatzke) [reply rules/cloud/aws_ec2_vm_export_failure.yml:22]: That's ok

**Comment by thomaspatzke** [comment]
Need to add the ci tests before merging.

**Comment by pdr9rc** [comment]
> Need to add the ci tests before merging.

@thomaspatzke Can you point me to what integration tests  you want me to add to? The 3 test scripts I saw running and asked you for clarification about are a little hard to understand to be honest. I would like to be able to run the tests successfully before writing anything else and since it isn't unit tests you seem to be asking for, I'm at a loss for an example of what sort of tests you'd want my changes to pass.

**Pull request merged**